flake-modules/dev/default.nix

```diff
@@ -24,6 +24,46 @@
         };
       };
 
+      packages = {
```

Comment:
I think it's better to use `apps` for stuff that we want to _run_ but don't normally need to _build_.

Flake-parts [takes care of](https://flake.parts/options/flake-parts.html#opt-perSystem.apps) the weirdness with the `apps` schema, so we can just set `apps.<name>.program` to a derivation (so long as it has a `meta.mainProgram`).